mne/viz/backends/_ipyvolume.py

```diff
@@ -0,0 +1,372 @@
+"""Core visualization operations based on ipyvolume."""
+
+# Authors: Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#          Eric Larson <larson.eric.d@gmail.com>
+#          Oleh Kozynets <ok7mailbox@gmail.com>
+#          Guillaume Favelier <guillaume.favelier@gmail.com>
+#
+# License: Simplified BSD
+
+import warnings
+import numpy as np
+from .base_renderer import _BaseRenderer
+from ...utils import copy_base_doc_to_subclass_doc
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", category=DeprecationWarning)
+    import ipyvolume as ipv
+    from pythreejs import (BlendFactors, BlendingMode, Equations,
+                           ShaderMaterial, Side)
+
+
+@copy_base_doc_to_subclass_doc
+class _Renderer(_BaseRenderer):
```

Comment:
Because `_Renderer` is the name we chose as the 3d backend-agnostic layer used already in `_pysurfer_mayavi.py` and `_pyvista.py`. `IPVRenderer` might be confusing as it suggests it's an `ipyvolume`-specific class.

Reply:
yes I saw it later in my review. I forgot to scratch it.